styles/css/vtex.product-summary.css

```diff
@@ -16,3 +16,17 @@
     display: none;
   } 
 }
+
+.nameContainer {
+  justify-content: left;
```

Comment:
[You can't use this value](https://caniuse.com/#search=justify-content%3A%20left), maybe is better to use just a `justify-content: start`

Reply:
`border-radius: 100%` didn't work out well